ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/ip_nat.schema.yml

```diff
@@ -0,0 +1,182 @@
+        primary_key: name
+        convert_types:
+          - dict
+        items:
+          type: dict
+          keys:
+            name:
+              type: str
+            prefix_length:
+              type: int
+              required: true
+              convert_types:
+                - str
+              min: 16
+              max: 32
+            ranges:
+              type: list
+              items:
+                type: dict
+                keys:
+                  first_ip:
+                    type: str
+                    required: true
+                    description: IPv4 address
+                  last_ip:
+                    type: str
+                    required: true
+                    description: IPv4 address
+                  first_port:
+                    type: int
+                    convert_types:
+                      - str
+                    min: 1
+                    max: 65535
+                  last_port:
+                    type: int
+                    convert_types:
+                      - str
+                    min: 1
+                    max: 65535
+            utilization_log_threshold:
+              type: int
+              convert_types:
+                - str
+              min: 1
+              max: 100
+      synchronization:
+        type: dict
+        keys:
+          description:
+            type: str
+          expiry_interval:
+            type: int
+            convert_types:
+              - str
+            min: 60
+            max: 3600
+            description: in seconds
+          local_interface:
+            type: str
+            description: EOS interface name
+          peer_address:
+            type: str
+            description: IPv4 address
+          port_range:
+            type: dict
+            keys:
+              first_port:
+                type: int
+                convert_types:
+                  - str
+                min: 1024
+                max: 65535
+              last_port:
+                type: int
+                convert_types:
+                  - str
+                min: 1024
+                max: 65535
+                description: '>= first_port'
+              split_disabled:
+                type: bool
+          shutdown:
+            type: bool
+      translation:
+        type: dict
+        keys:
+          address_selection:
+            type: dict
+            keys:
+              any:
+                type: bool
+              hash_field_source_ip:
+                type: bool
+          counters:
+            type: bool
+          low_mark:
+            type: dict
+            keys:
+              percentage:
+                type: int
+                convert_types:
+                  - str
+                min: 1
+                max: 99
+              host_percentage:
+                type: int
+                convert_types:
+                  - str
+                min: 1
+                max: 99
```

Comment:
Maybe for this one because the key can look "surprising" for someone used to the CLI you can add a description stating
```
Used to render 'ip nat translation low-mark <host_percentage> host
```

(Happy to be told no on this one - just wondering if this would be obvious for someone coming to AVD)

Reply:
I can add the comment. No problem.

Reply:
Fixed in [8899c99](https://github.com/aristanetworks/ansible-avd/pull/2747/commits/8899c9909d24d56209fec75f70e38110c176d56b)